src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj

```diff
@@ -25,6 +25,7 @@
     <AssemblyName>Microsoft.Web.LibraryManager.Vsix</AssemblyName>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
+    <TargetVsixContainerName>Microsoft.Web.LibraryManager.Vsix</TargetVsixContainerName>
```

Comment:
nit: the file extension should be lower case for consistency

Reply:
Missed that...